.github/workflows/release-dev.yml

```
@@ -1,15 +1,14 @@
-name: Create Github pre-release
+name: Create Github Dev Release
 
 on:
   workflow_dispatch:
     inputs:
       taipy-config-version:
-        description: "The taipy-config version to use (ex: 2.3.0.dev0)"
-        required: true
+        description: "The taipy-config version to use (ex: 1.0.0.dev0)"
 
 jobs:
-  publish:
-    timeout-minutes: 20
+  test-release-package:
```

I don't understand the prefix test. Why not:
```suggestion
  release-dev-package:
```